internal/pkg/manifest/validate_test.go

```diff
@@ -565,6 +565,37 @@ func TestBackendService_validate(t *testing.T) {
 			},
 			wantedError: errors.New(`"http" must be specified if "count.requests" or "count.response_time" are specified`),
 		},
+		"error if invalid topic is defined": {
+			config: BackendService{
+				BackendServiceConfig: BackendServiceConfig{
+					ImageConfig: testImageConfig,
+					PublishConfig: PublishConfig{
+						Topics: []Topic{
+							{
+								Name: aws.String("mytopic.fifo"),
+							},
+						},
+					},
+				},
+			},
+			wantedErrorMsgPrefix: `validate "publish": `,
+		},
+		"error if invalid type is defined": {
+			config: BackendService{
+				BackendServiceConfig: BackendServiceConfig{
+					ImageConfig: testImageConfig,
+					PublishConfig: PublishConfig{
+						Topics: []Topic{
+							{
+								Name: aws.String("mytopic"),
+								Type: aws.String("incorrectValue"),
+							},
+						},
+					},
+				},
+			},
+			wantedErrorMsgPrefix: `validate "publish": `,
```

Comment:
Is there a way of writing a test case that validates the content of the new error message? maybe:
```suggestion
			wantedErrorMsgPrefix: `validate "publish": validate topics[0]: "type" value "incorrectValue" is not allowed`,
```